dockers/docker-lldp/lldpmgrd

```diff
@@ -218,6 +277,8 @@ class LldpManager(daemon_base.DaemonBase):
 
             # Process all pending commands
             self.process_pending_cmds()
+            if state == swsscommon.Select.TIMEOUT:
+                self.lldp_check_mgmt_info_change()
```

Comment:
it is unclear why polling mode is used here instead of existing subscribe mode.

Reply:
Yeah we need to change this to subscribe mode, when this change was made notifications were not received by lldpmgrd when these parameters were modified, so had to resort to using poll mode.

Reply:
@sandeep-kulambi Not sure what is it mean by this . You will change to use subscriberStateTable or you used and it did not work ?

Reply:
@abdosi - we will change this to subscribe mode